src/canmatrix/compare.py

```diff
@@ -568,22 +587,25 @@ def main():
         ignore["VALUETABLES"] = True
 
     if cmdlineOptions.frames:
-        onlyInMatrix1 = []
-        onlyInMatrix2 = []
+        only_in_matrix1 = []  # type: typing.List[str]
```

Comment:
https://play-mypy.ymyzk.com/?gist=8a77deb4f961f0162ab83f5fddc826be
It looks like comprehensions work well.  Also mypy seems to take the first added type as the intended element type anyways.

```suggestion
        only_in_matrix1 = [
            frame.name
            for frame in db1.frames
            if db2.frame_by_name(frame.name) is None
        ]
```